explainer.md

```diff
@@ -78,15 +78,33 @@ Icons will be registered for each handler based on the optional `icons` entry, w
 
-Choosing to open the file would create a new top level browsing context, navigating to the `action` URL (resolved against the manifest URL). Assuming the user opened `graph.csv` in Grafr the URL would be `https://grafr.com/open-csv/`. 
-
 ## Launch
 
-To access launched files, a site should specify a consumer for a `launchQueue` object attached to `window`. Launches are queued until they are handled by this consumer, which is invoked exactly once for each launch. In this manner, we can ensure every launch is handled, regardless of when the consumer was specified. The application will have read access through the [File System Access](https://github.com/WICG/file-system-access/blob/master/EXPLAINER.md) API, but will need to request write access from the user in order to edit the file.
+Choosing to open the file would create a new top level browsing context, navigating to the `action` URL (resolved against the manifest URL). Assuming the user opened `graph.csv` in Grafr the URL would be `https://grafr.com/open-csv/`. 
 
-> Note: The `launchParams` property is discussed in more detail in the [Launch Events](https://github.com/WICG/sw-launch/issues/20) explainer.
+To access launched files, a site should specify a consumer for a `launchQueue` object attached to `window`. Launches are queued until they are handled by this consumer, which is invoked exactly once for each launch. In this manner, we can ensure every launch is handled, regardless of when the consumer was specified. The application will have read access through the [File System Access](https://github.com/WICG/file-system-access/blob/master/EXPLAINER.md) API, but will need to separately request write access from the user to edit the file.
 
-Below is a basic example receiving the file handles.
+`launchQueue`, `launchParams`, and the DOM Window they're used on, are specified as follows:
+```
+// DOM Window Launch Queue.
+partial interface Window {
+    readonly attribute LaunchQueue launchQueue;
+};
+
+// Launch Queue.
+[Exposed=Window] interface LaunchQueue {
+  void setConsumer(LaunchConsumer consumer);
+};
+
+callback LaunchConsumer = any (LaunchParams params);
+
+// Launch Params.
+[Exposed=Window] interface LaunchParams {
+  readonly attribute FrozenArray<FileSystemHandle> files;
+};
+```
+
+Below is a basic example using `launchQueue` and `launchParams` to recieve the file handles.
```

Comment:
typo: receive

I think you want "Below is a basic example of using"

Reply:
oof thanks. I think I need to proofread my own commits more... I'm too used to spellcheckers these days...